cmd/cmount/mount.go

```diff
@@ -180,6 +181,10 @@ func Mount(f fs.Fs, mountpoint string) error {
 	sigHup := make(chan os.Signal, 1)
 	signal.Notify(sigHup, syscall.SIGHUP)
 
+	if err := sdnotify.SdNotifyReady(); err != nil && err != sdnotify.SdNotifyNoSocket {
+		return errors.Wrap(err, "failed to notify systemd")
```

Comment:
I'd say this probably just wants to be a `fs.Infof` at worst rather than returning an error which stops the mount process - the system in use may not have systemd or have the environment variable set.

Reply:
Also can this code go in mountlib/mount.go so you don't have to duplicate it?

Reply:
I made the initial notify error fatal, because when `NOTIFY_SOCKET` is set, systemd is waiting for the `READY=1` event. Without it the process will get stuck in the starting phase and restarted when`TimeoutStartSec` is reached.
An alternative would be adding a `--notify` command line argument, which enables the notification and returns a fatal error on failure.

Reply:
I couldn't find a place to put the notification in mountlib. It has to trigger after the `func mount()` call cmount and mount so that the mount point is populated.

Reply:
I was just looking at the [code for Sdnotify](https://github.com/okzk/sdnotify/blob/master/notify_linux.go#L9-L12) and I noticed that it returns
 `sdnotify.SdNotifyNoSocket`.. I see you are checking for that which I missed before!

I take your point about mountlib - what you've done looks fine.